CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## vNext
 
+- Implement transfer tx links
```

Comment:
1. Use perfective verb
2. Add more details to the feature but keep the description as small as possible.